dashboard/app/models/contact_rollups_pardot_memory.rb

```diff
@@ -95,6 +95,23 @@ def self.download_pardot_prospects(last_id = nil, limit = nil)
     end
   end
 
+  def self.download_deleted_pardot_prospects(last_id = nil, limit = nil)
+    PardotV2.retrieve_prospects(last_id || 0,  %w(id email), limit, true) do |deleted_prospects|
```

Comment:
Just curious...I think I had set 0 as the default instead of nil, thinking that `last_id` would either be 0 or the provided argument. What you've done here maybe seems like better style though, to show that in `retrieve_prospects` the first argument is either the provided last_id or 0?

Reply:
I set the default value to `nil` for a couple of reasons
1. To be consistent with other `download_*` methods in this class.
2. Both `last_id` and `limit` are optional parameters, if I just want to set a `limit` value (without setting `last_id` value), I can write `PardotV2.download_deleted_pardot_prospects(nil, 10)`.

Thinking about this a bit more, maybe we don't need to use `last_id` value of 0 at all. Will remove it.